samples/drivers/fpga/fpga_controller/README.rst

```diff
@@ -113,3 +113,12 @@ Now the bitstream can be uploaded again.
 
     uart:~$ fpga load FPGA 0x10000 75960
     FPGA: loading bitstream
+
+References
+**********
+
+.. _Quicklogic Quickfeather board:
+   https://github.com/QuickLogic-Corp/quick-feather-dev-board
+
+.. _QuickFeather programming and debugging:
+   https://docs.zephyrproject.org/latest/boards/arm/quick_feather/doc/index.html#programming-and-debugging
```

Comment:
we should not be hardcoding links to our own docs, use references instead.

Reply:
That also oçurred to me but i'm no .rst guru. Any idea what that would look like? I have no idea.